TESTS/configs/baremetal_std.json

```diff
@@ -0,0 +1,42 @@
+{
+    "requires": [
```

Comment:
This section is duplicated between `baremetal_std.json` and `baremetal_small.json`. Is there a way to avoid the duplication as this will create maintenance issues?